fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -938,7 +937,7 @@ public void testGetRepositoryGraph() throws IOException {
         try (final CloseableGraphStore graph = getGraphStore(getObjMethod(""))) {
             logger.trace("Retrieved repository graph:\n" + graph);
             assertTrue("Should find the root type", graph.contains(ANY,
-                    ANY, HAS_PRIMARY_TYPE.asNode(), createLiteral(ROOT)));
+                    ANY, type.asNode(), createURI(MODE_NAMESPACE + "root")));
```

Comment:
We could add a type (not a property). I don't know that we even need to do that. The inference is pretty straightforward. If you are curious as to whether a thing is the repo root, check to see if it has a parent. If so, no, it isn't. If not, yes, it is.